admin_guide/out_of_resource_handling.adoc

```diff
@@ -174,31 +159,26 @@ If an eviction threshold is met and the grace period is passed, the node
 initiates the process of evicting pods until it observes the signal going below
 its defined threshold.
 
-The node ranks pods for eviction:
+The node ranks pods for eviction by their
+xref:../admin_guide/overcommit.adoc#qos-classes[quality of service], and, among
+those with the same quality of service, by the consumption of the starved
+compute resource relative to the pod's scheduling request.
 
-. By their xref:../admin_guide/overcommit.adoc#qos-classes[quality of service]
-(`BestEffort`, `Burstable`, or `Guaranteed`), and
-. Among those with the same quality of service by the consumption of the
-starved compute resource relative to the pod's scheduling request.
-
-* `BestEffort` pods that consume the most of the starved resource are failed
+* `BestEffort`: pods that consume the most of the starved resource are failed
 first.
-* `Burstable` pods that consume the greatest amount of the starved resource
-relative to their request for that resource are killed first. If no pod has
-exceeded its request, the strategy targets the largest consumer of the starved
-resource.
-* `Guaranteed` pods that consume the greatest amount of the starved resource
-relative to their request are killed first. If no pod has exceeded its request,
+* `Burstable`: pods that consume the most of the starved resource relative to their
+request for that resource are failed first. If no pod has exceeded its request,
 the strategy targets the largest consumer of the starved resource.
+* `Guaranteed`: pods that consume the most of the starved resource relative to
+their request are killed first. If no pod has exceeded its request, the strategy
+targets the largest consumer of the starved resource. 
 
 A `Guaranteed` pod is guaranteed to never be evicted because of another pod's
-resource consumption.
-
-If:
+resource consumption if:
```

Comment:
The if is confusing here.

A `Guaranteed` pod is guaranteed to never be evicted as a result of another pod's resource consumption.

The next section is trying to say:

The only reason a `Guaranteed` pod could be evicted is as follows:

* a system daemon...
* ...